kubernetes-aws-python/__main__.py

```diff
@@ -0,0 +1,40 @@
+import pulumi
+import pulumi_awsx as awsx
+import pulumi_eks as eks
+
+# Get some values from the Pulumi configuration (or use defaults)
+config = pulumi.Config()
+min_cluster_size = 3 if config.get_float("minClusterSize") is None else config.get_float("minClusterSize")
+max_cluster_size = 6 if config.get_float("maxClusterSize") is None else config.get_float("maxClusterSize")
+desired_cluster_size = 3 if config.get_float("desiredClusterSize") is None else config.get_float("desiredClusterSize")
+eks_node_instance_type = "t2.medium" if config.get("eksNodeInstanceType") is None else config.get("eksNodeInstanceType")
+vpc_network_cidr = "10.0.0.0/16" if config.get("vpcNetworkCidr") is None else config.get("vpcNetworkCidr")
```

Comment:
Haha, it was @nimbinatus that suggested the current syntax! 🤣

Reply:
... I just realized that the get methods for Pulumi config are actually using the dictionary get methods 🤦‍♀️  Sorry! The version I was doing was for general default values (it handles falsey values differently than a simple or). I thought they had made new get methods instead of using the underlying method, which actually can handle default values natively. https://www.pulumi.com/docs/reference/pkg/python/pulumi/#pulumi.Config.get_float New syntax:

```python
min_cluster_size = config.get_float("minClusterSize", 3)
max_cluster_size = config.get_float("maxClusterSize", 6)
desired_cluster_size = config.get_float("desiredClusterSize", 3)
eks_node_instance_type = config.get("eksNodeInstanceType", "t2.medium")
vpc_network_cidr = config.get("vpcNetworkCidr", "10.0.0.0/16")
```

Reply:
Oh, nice! I didn't know this worked for Python. Much nicer. Thanks @nimbinatus!

Reply:
Fixed in 8f2808f